.yaydoc.yml

```diff
@@ -0,0 +1,13 @@
+metadata:
+  projectname: Open Event Organizer App
+  author: FOSSASIA
+  version: development
+
+build:
+  source: docs
+  theme:
+    name: sphinx_fossasia_theme
+
+extras:
+  javadoc:
+    path: app/src
```

Comment:
since javadoc is not working for the android app. I think there is no necessary of having this PR. so please close this PR